checkov/common/bridgecrew/platform_integration.py

```diff
@@ -572,7 +573,7 @@ def commit_repository(self, branch: str) -> str | None:
                                                                 get_user_agent_header()
                                                                 ))
                 response = json.loads(request.data.decode("utf8"))
-                url: str = response.get("url", None)
+                url: str = self.get_sso_prismacloud_url(response.get("url", None))
```

Comment:
@SimOnPanw what happens if SSO isn't configured?

Reply:
Hello @mikeurbanski1, we will be returning the standard URL that directly points to the platform. To return the URL with RelayState parameter,  you'll have to provide a prisma_api_url and ensure that your Prisma Cloud tenant is configured with both relay_state_param_name and access_saml_url.

Here is an example:
![image](https://github.com/bridgecrewio/checkov/assets/91271604/24eaec59-6b14-4ce2-aa47-5a9da8484fdc)

Reply:
Hello @mikeurbanski1 and @gruebel, any news on this PR ?

Reply:
But what does `{bc_integration.prisma_api_url}/saml/config` return if SSO is not configured? It looks like we are always going to try to get this, and throw an exception if it doesn't work, so I am worried that we won't fall back.

Reply:
It returns a json with an empty fields for relayStateParamName and redLockAccessSamlUrl, even if a tenant has no SAML configuration.
I checked if there is any value return below in the code.